src/lib/button/button.scss

```diff
@@ -24,11 +24,12 @@
 }
 
 .mat-fab {
-  @include mat-fab($mat-fab-size, $mat-fab-padding);
+  @include mat-fab($mat-fab-size);
 }
 
 .mat-mini-fab {
-  @include mat-fab($mat-mini-fab-size, $mat-mini-fab-padding);
+  @include mat-fab($mat-mini-fab-size);
+  font-size: 80%;
```

Comment:
I don't see anything in the spec that talks about font-size for the mini fab- what are you basing this on?

Reply:
Basing on it was unproportionate in the smaller use case. Happy to remove if you would like.

Reply:
Removed.